frontend/src/pages/instance/Settings/LauncherSettings.vue

```diff
@@ -8,6 +8,13 @@
                 Flows that perform CPU intensive work may need to increase this from the default of 7500ms.
             </template>
         </FormRow>
+        <FormRow v-model="input.disableAutoSafeMode" type="checkbox">
+            Disable Auto Safe Mode
+            <template #description>
+                Prevent Node-RED from automatically entering safe mode when a crash loop is detected.
+                WARNING: Disabling Auto Safe Mode is not recommended. A problem that causes Node-RED to crash multiple successive times may result in a contineous bootloop that will need to be manually resolved.
+            </template>
+        </FormRow>
```

Comment:
This is going to need a launcher version check so it's not shown to pre 2.12 launchers and a message about upgrading is shown.

See here for an example 
- https://github.com/FlowFuse/flowfuse/blob/main/frontend/src/pages/admin/Template/sections/NPMRegistry.vue
- https://github.com/FlowFuse/flowfuse/pull/4900/files#diff-b9045860ec8e01709bcc1244f38e95637e7dfdb7df7dd4e114954506137bcb16

Reply:
> This is going to need a launcher version check so it's not shown to pre 2.12 launchers and a message about upgrading is shown.

ah, indeed. good call Ben.

Will push update soon.